spec/student_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../student'
+
+# rubocop:disable Lint/UselessAssignment
+
+describe Student do
+  before(:each) do
+    @student = Student.new('student', 5, 'Tiwa', parent_permission = false)
+  end
+
+  it 'Should be create a student name' do
+    expect(@student.name).to eq 'Tiwa'
+  end
+  it 'Should be create a stydent age' do
+    expect(@student.age).to eq 5
+  end
+
+  it 'Should be create a student classroom' do
+    expect(@student.classroom).to eq 'student'
+  end
+
+  it 'Should be create a student check student parent permission' do
+    expect(@student.parent_permission).to eql false
+  end
+end
+
+# rubocop:enable Lint/UselessAssignment
```

Comment:
- [ ] Same here, Please let's make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉 

- [ ] There should be unit tests for the student classes, Kindly make sure to test all methods of the student class !!👇🏻 check your student,rb file


```ruby
 # Method play_hooky
  def play_hooky
    "¯\(ツ)/¯"
  end

Reply:
Already implemented it! Thanks for the feedback! :smile: